browser/Makefile.am

```diff
@@ -226,6 +226,7 @@ COOL_JS_LST =\
 	src/layer/tile/TilesSection.ts \
 	src/layer/tile/AutoFillMarkerSection.ts \
 	src/layer/tile/CellCursorSection.ts \
+	/src/layer/tile/HTMLObjectSection.ts \
```

Comment:
shouldn't this be `src/layer/tile/HTMLObjectSection.ts \` (no starting `/` ?

Reply:
Thank you for checking.